### PR TITLE
Bug 1827194: allow using VM Template with http/url dataVolumeTemplates

### PR DESCRIPTION
**Pull request by suomiy**
- as a source by VMWizard
- make better format of disk name (also compatible with vm-import-operator)

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/storage-tab/storage-boot-source.tsx [hunk 1]
```diff
@@ -26,14 +26,14 @@ export const StorageBootSource: React.FC<StorageBootOrderProps> = ({
...
-  const filteredStorages = storages.filter(({ volume, dataVolume }) =>
-    [StorageUISource.ATTACH_DISK, StorageUISource.ATTACH_CLONED_DISK].includes(
-      StorageUISource.fromTypes(
-        new VolumeWrapper(volume).getType(),
-        dataVolume && new DataVolumeWrapper(dataVolume).getType(),
-      ),
-    ),
-  );
+  const filteredStorages = storages.filter(({ volume, dataVolume }) => {
+    return (
```
Comment (yaacov) [inline frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/storage-tab/storage-boot-source.tsx:30]: why use return in this function ?

Comment (suomiy) [reply frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/storage-tab/storage-boot-source.tsx]: changed

**Pull request merged**